lib/gtfs_realtime_viz/state.ex

```diff
@@ -52,10 +102,54 @@ defmodule GTFSRealtimeViz.State do
       end
     end)
 
-    {:reply, :ok, new_state}
+    {:reply, :ok, %{state | vehicles: new_vehicles}}
   end
   def handle_call({:vehicles, group}, _from, state) do
-    {:reply, state[group], state}
+    case state.vehicles[group] do
+      nil -> {:reply, [], state}
+      vehicles -> {:reply, vehicles, state}
+    end
+  end
+
+  def handle_call({:trip_updates, group, trip_updates, comment}, _from, state) do
```

Comment:
I don't think anything calls this anymore does it?

Reply:
true, i guess this and the vehicles equivalent are now unnecessary.